plugins/frontend/backstage-plugin-argo-cd/src/api/index.ts

```diff
@@ -158,67 +181,86 @@ export class ArgoCDApiClient implements ArgoCDApi {
         options.app as string,
       )}/revisions/${encodeURIComponent(
         options.revisionID as string,
-      )}/metadata`,
+      )}/metadata${query}`,
       argoCDAppDeployRevisionDetails,
     );
   }
 
   async getAppDetails(options: {
     url: string;
     appName: string;
+    appNamespace?: string;
     instance?: string;
   }) {
     const proxyUrl = await this.getBaseUrl();
+    const query = this.getQueryParams({
+      appNamespace: options.appNamespace,
+    });
     if (this.searchInstances) {
       return this.fetchDecode(
         `${proxyUrl}/argoInstance/${
           options.instance
-        }/applications/name/${encodeURIComponent(options.appName as string)}`,
+        }/applications/name/${encodeURIComponent(
+          options.appName as string,
+        )}?${query}`,
```

Comment:
The ? before ${query} is wrong here, isn't it? the getQueryParams will add the ? automatically.

If yes, please remove.

Reply:
Good catch, fixed it